e2e/testdata/fn-render/generator-subpkgs/.expected/diff.patch

```diff
@@ -1,25 +1,44 @@
-diff --git a/db/configmap_db-map.yaml b/db/configmap_db-map.yaml
-new file mode 100644
-index 0000000..7a0f3af
---- /dev/null
-+++ b/db/configmap_db-map.yaml
-@@ -0,0 +1,6 @@
-+apiVersion: v1
-+kind: ConfigMap
-+metadata:
-+  name: db-map
-+data:
-+  db-key: db-value
-diff --git a/def/configmap_parent-map.yaml b/def/configmap_parent-map.yaml
-new file mode 100644
-index 0000000..fd1e17d
---- /dev/null
-+++ b/def/configmap_parent-map.yaml
-@@ -0,0 +1,7 @@
-+apiVersion: v1
-+kind: ConfigMap
-+metadata:
-+  name: parent-map
-+  namespace: def
-+data:
-+  parent-key: parent-value
+diff --git a/db/fn-config.yaml b/db/fn-config.yaml
+index 66b0d31..7a0f3af 100644
+--- a/db/fn-config.yaml
++++ b/db/fn-config.yaml
+@@ -1,17 +1,3 @@
+-# Copyright 2021 Google LLC
```

Comment:
It seems the license header become part of the diff. It's verbose and not useful.
I guess we don't want the license header to be part of the diff.

Reply:
Removed the license header.